presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -45,3 +45,101 @@ which is configured with a carefully selected set of options. This behavior
 can be overridden by setting the environment variable ``PRESTO_PAGER`` to the
 name of a different program such as ``more`` or `pspg <https://github.com/okbob/pspg>`_,
 or it can be set to an empty value to completely disable pagination.
+
+Output Formats
+--------------
+
+The Presto CLI provides the option ``--output-format`` to control how the output
+is displayed when running in noninteractive mode. The available options shown in
+the following table must be entered in uppercase. The default value is ``CSV``.
+
+.. list-table::
+  :widths: 20, 80
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``CSV``
+    - Comma-separated values, each value quoted. No header row.
+  * - ``CSV_HEADER``
+    - Comma-separated values, quoted with header row.
+  * - ``CSV_UNQUOTED``
+    - Comma-separated values without quotes.
+  * - ``CSV_HEADER_UNQUOTED``
+    - Comma-separated values with header row but no quotes.
+  * - ``TSV``
+    - Tab-separated values.
+  * - ``TSV_HEADER``
+    - Tab-separated values with header row.
+  * - ``JSON``
+    - Output rows emitted as JSON objects with name-value pairs.
+  * - ``ALIGNED``
+    - Output emitted as an ASCII character table with values.
+  * - ``VERTICAL``
+    - Output emitted as record-oriented top-down lines, one per value.
+  * - ``NULL``
+    - Suppresses normal query results. This can be useful during development
+      to test a query's shell return code or to see whether it results in
+      error messages.
+
+Examples
+^^^^^^^^
+
+Consider the following command run as shown, or with ``--output-format CSV``:
+
+.. code-block:: none
+
+   presto --execute 'SELECT nationkey, name, regionkey FROM tpch.sf1.nation LIMIT 3'
+
+The output is:
+
+.. code-block:: none
+
+  "0","ALGERIA","0"
+  "1","ARGENTINA","1"
+  "2","BRAZIL","1"
+
+With ``--output-format JSON``:
+
+.. code-block:: json
+
+  {"nationkey":0,"name":"ALGERIA","regionkey":0}
+  {"nationkey":1,"name":"ARGENTINA","regionkey":1}
+  {"nationkey":2,"name":"BRAZIL","regionkey":1}
+
+With ``--output-format ALIGNED``:
+
+.. code-block:: none
+
+   nationkey |   name    | regionkey
+   ----------+-----------+----------
+           0 | ALGERIA   |         0
+           1 | ARGENTINA |         1
+           2 | BRAZIL    |         1
+
+With ``--output-format VERTICAL``:
+
+.. code-block:: none
+
+  -[ RECORD 1 ]--------
+  nationkey | 0
+  name      | ALGERIA
+  regionkey | 0
+  -[ RECORD 2 ]--------
+  nationkey | 1
+  name      | ARGENTINA
+  regionkey | 1
+  -[ RECORD 3 ]--------
+  nationkey | 2
+  name      | BRAZIL
+  regionkey | 1
+
+The preceding command with ``--output-format NULL`` returns 0, and no output
```

Comment:
Saying "returns 0" here is confusing, since "return" can mean output, and we're talking about output formats. Since that's the default and standard exit status, I'm not sure mentioning it is helpful. How about something like
```
The preceding command with ``--output-format NULL`` produces no output.
However, if you have an error in the query, such as incorrectly using
``region`` instead of ``regionkey``, the command has an exit status of 1
and displays an error message (which is unaffected by the output format):